tests/host/test_host_reverse.yml

```diff
@@ -26,15 +25,29 @@
     set_fact:
       ipv4_prefix: "{{ ansible_default_ipv4.address.split('.')[:-1] |
                        join('.') }}"
-      reverse_zone: "{{ ansible_default_ipv4.address.split('.')[2::-1] |
-                        join('.') }}"
+
+  - name: Set zone prefixes.
+    set_fact:
+      zone_ipv6_reverse: "ip6.arpa."
+      zone_ipv6_reverse_workaround: "d.f.ip6.arpa."
+      zone_prefix_reverse: "in-addr.arpa"
+      zone_prefix_reverse_8: "{{ ipv4_prefix.split('.')[2::-1] | join ('.') }}.in-addr.arpa"
+      zone_prefix_reverse_16: "{{ ipv4_prefix.split('.')[1::-1] | join ('.') }}.in-addr.arpa"
+      zone_prefix_reverse_24: "{{ ipv4_prefix.split('.')[::-1] | join ('.') }}.in-addr.arpa"
```

Comment:
These are bigger changes that are not related to `_domain` or `_realm` and should be in a separate commit.

Reply:
Ok. I fixed 4 bugs in a single commit. I'll create 1 commit for each fix.